frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -3,8 +3,8 @@
 action.GitMachete.description.disabled.undefined.branch-name={0} disabled due to undefined branch name
 action.GitMachete.description.disabled.undefined.machete-branch={0} disabled due to undefined machete branch
 
-GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec {0} failed
-GitMachete.FetchBackgroundable.notification.success=Fetch of refspec {0} succeeded
+action.GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec {0} failed
```

Comment:
`FetchBackgroundable` is not an action :confused:

Reply:
Hmmm it's true... but TBH shouldn't it then be `GitMachete.action.xxx` or `GitMachete.FetchBackgroundable.xxx`? Actually, do we even need that `GitMachete` prefix for anything? :thinking:

Reply:
We need `GitMachete` for bundles that are referenced by `plugin.xml` only

Reply:
_We need_ = _without that prefix, we could get a name conflict with a property of the same key defined somewhere else in the classpath_, right?

Reply:
Or is there any other reason?

Reply:
Also, replacing `action.GitMachete.xxx` with `GitMachete.action.xxx` everywhere should still be okay wrt. plugin.xml, right?